devops/jobs/SandboxTermination.groovy

```diff
@@ -0,0 +1,133 @@
+/*
+ Variables without defaults are marked (required) 
+ 
+ Variables consumed for this job:
+    * NOOP: boolean value to perform no operations, default is false
+    * SECURE_GIT_CREDENTIALS: secure-bot-user (required)
+    * SYSADMIN_REPO: repository containing sandbox termination python script (required)
+    * SYSADMIN_BRANCH: default is master
+    * CONFIGURATION_REPO: name of config repo, default is https://github.com/edx/configuration.git
+    * CONFIGURATION_BRANCH: default is master
+    * CONFIGURAITON_INTERNAL_REPO: Git repo containing internal overrides, default is git@github.com:edx/edx-internal.git
+    * CONFIGURATION_INTERNAL_BRANCH: default is master
+    * ROUTE53_ZONE: AWS route53 zone for getting DNS records (requried)
+    * AWS_REGION: region of running sandbox instances, default is us-east-1
+    * NOTIFY_ON_FAILURE: alert@example.com
+    * FOLDER_NAME: folder
+
+ This job expects the following credentials to be defined on the folder
+    tools-edx-jenkins-aws-credentials: file with key/secret in boto config format
+    launch-sandboxes-role-arn: the role to aws sts assume-role
+ 
+*/
+
+package devops.jobs
+import static org.edx.jenkins.dsl.Constants.common_logrotator
+import static org.edx.jenkins.dsl.Constants.common_wrappers
+
+class SandboxTermination{
+    public static def job = { dslFactory, extraVars ->
+        dslFactory.job(extraVars.get("FOLDER_NAME","Monitoring") + "/sandbox-termination") {
+            
+            logRotator common_logrotator
+            wrappers common_wrappers
+
+            wrappers{
+                credentialsBinding{
+                    file('AWS_CONFIG_FILE','tools-edx-jenkins-aws-credentials')
+                    string('ROLE_ARN', "launch-sandboxes-role-arn")
+                }
+                sshAgent("ubuntu_deployment_201407")
```

Comment:
@jibsheet yeah I just copied it over from admin. I tested the job without it, and it worked so I'll remove it